### PR TITLE
gpl: modify log and readme for timing driven mode clarity

### PR DESCRIPTION
**Pull request by gudeh**
Include more clear messages during timing driven mode so the user can better understand what is happening. 
Include possible actions for user to improve runtime due to slow timing driven mode.
Include new tcl command in README for using grt(previous) routing congestion estimator during routability.
Fix default inflation parameters in README.

### CODE REVIEW

**Comment by gudeh** [comment]
These changes should address issue #5203.

**Comment by gudeh** [comment]
I included the message "Timing-driven: executing resizer for reweighting nets." because the resizer may take a long time, this way the user can know why the tool is loading.

**Comment by maliberty** [comment]
@oharboe please comment as it is your issue

File: src/gpl/README.md [hunk 1]
```diff
@@ -34,17 +34,24 @@ weight nets with low slack. It adjusts the worst slacks (modified with
...
-attempt to achieve the target RC (routing congestion) by comparing it to the 
-final RC. If the algorithm takes too long during routability-driven execution, 
-consider raising the target RC value to alleviate the constraints. The final 
-RC value is calculated based on the weight coefficients. The algorithm will 
-stop if the RC is not decreasing for three consecutive iterations.
+execute RUDY to provide an estimation of routing congestion. Congested tiles 
+will have the area of their logic cells inflated to reduce routing congestion. 
+The iterations will attempt to achieve the target RC (routing congestion) 
+by comparing it to the final RC at each iteration. If the algorithm takes too 
+long during routability-driven execution, consider raising the target RC value
```
Comment (oharboe) [inline src/gpl/README.md:51]: as above. Reduce guesswork; be explicit about which option to use to "the target RC value".

**Review by oharboe — CHANGES_REQUESTED** [review]
Just a nit, see comments.

More general comments:

I think the main usability improvement here is that RUDY is used and that is adequately documented.

The other routing options require a strong mental model of what is going on. They are not "user-servicable parts" to my mind.

A user-servicable knob would be something like a value between 0 and 1 where 0 is fastest and 1 is best quality of results...

I'm not suggesting that such a feature should be implemented, but are interested in hearing thoughts.

I think the explanation of the "non user-servicable parts" of OpenROAD here are good.

**Comment by gudeh** [comment]
I realized we do not have the number of buffers inserted (and other modifications by resizer) in the gpl log. Which makes sense because we do a virtual resizing. I will include this info in a new PR, since it is more related to issue #5291.

Do you think this modifications are good to go? @maliberty @oharboe

**Review by oharboe — APPROVED** [review]
This level of detail on global router is too deep for my current understanding, so I can't comment much.

**Review by maliberty — APPROVED** [review]

**Comment by maliberty** [comment]
> I realized we do not have the number of buffers inserted (and other modifications by resizer) in the gpl log.

This may be better as a debugPrint as I'm not sure users will care.

**Pull request merged**